packages/shared/src/utils/constants/jurisdictions-config.ts

```diff
@@ -0,0 +1,7 @@
+export const Jurisdiction = Object.freeze({
```

Comment:
@amina-deriv Let's not do the same thing as we did with the `ContentFlag` constant 🙏🏻 

The issue with `ContentFlag` was it's been used in so many places with many different `.includes` conditions which made the logic not maintainable, After a while as we did bug fixes we ended up with different `.includes` for the same thing and now no one knows which is the correct one 😅

You guys can create hooks instead of importing this variable everywhere and doing `.includes` check to hide the logic from devs so they will be just using the hook and if something change later we would only need to fix it in one place (and we can write test for the hook) 🙇🏻 

```ts
const useSelectedJurisdiction = () => {
    // Todo: Should move the `jurisdiction_selected_shortcode` property to one of the core stores
    // like `client_store` as later we will be removing `modules` from the stores.
    const { modules } = useStore();
    const { cfd } = modules;
    const selected_jurisdiction = cfd.jurisdiction_selected_shortcode;
    const is_svg = selected_jurisdiction === 'svg';
    const is_bvi = selected_jurisdiction === 'bvi';
    const is_vanuatu = selected_jurisdiction === 'vanuatu';
    const is_labuan = selected_jurisdiction === 'labuan';
    const is_maltainvest = selected_jurisdiction === 'maltainvest';

    return {
        is_svg,
        is_bvi,
        is_vanuatu,
        is_labuan,
        is_maltainvest,
        is_vanuatu_or_is_maltainvest: is_vanuatu || is_maltainvest,
        is_something_or_something_else_and_some_other_thing: is_svg || (is_bvi && is_vanuatu),
    };
};
```

Reply:
@matin-deriv Please advice if we should consider this change or we can just ignore it as it will affect almost half of the PR 🙇🏻 🙏🏻

Reply:
@farzin-deriv `Jurisdiction` is just a constant object to help us keep all the constants in one place so it is easier to update them in future.

Reply:
@matin-deriv I'd rely on TS to protect the object from mutations since it's a TS file, so I suggest that we use as const instead of Object.freeze()
```
export const Jurisdiction = {
    ...
} as const;
```
But I believe it's a matter of taste 😄